### DISCUSSION TITLE
Question 67: RRC Website, Collector Number [ZK, NH - Action]

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#643] [Q&A]
Description: It’s my understanding that the RRC website pulls the data for Collector Number from the Collection Events record and that this field needs to be filled in on the CE record in order for it to show up on the RRC website. We’re thinking about how we should be setting up our CEs and in this scenario we would need a unique CE for every catalog record since every CE record would have its own Collector Number. I know that this is the way it’s done right now, but we were thinking we would want to be able to use a single CE record for multiple catalog records in the future. Is this the way we would want to set it up? And in this case would it be possible to have the RRC website pull Collector Number from the catalog module instead of the CE module? That way would it show up on the RRC site and also we (eventually) wouldn’t need a unique CE for every catalog record, or is there another way I should be thinking about this?

**Comment by zkachian** [comment]
A clarification on this: here we are referring to Collection Number, which has a field in catalogue called "Collection Number," [ColMammalsCollectionNo] and all our plant records use that field. - no need to put in OtherNumbers, and not the same as "CatalogNo." 
Alternatively we could use Botany's collection number field [ColCollectionNumber].  We wanted to see if it would be possible to pull this info from either of these fields in the Catalogue module, instead of the Collection Events module where it is currently pulled from, for the RRC website?

**Reply by rondlg** [reply]
In the past Botany did not use collection events. However, this was not a standard workflow and that was corrected by autofilling the collectionevent from the catalogue for Botany (and RRC).

In order for websites to be created efficiently data is pulled from standard fields in the case of collection number this is from the collectionevent field.

![image](https://user-images.githubusercontent.com/702709/193680394-f333ebe8-72c3-42e3-91f0-f189dc56160a.png)
